src/main/kotlin/net/atos/zac/mail/MailMessageBuilder.kt

```diff
@@ -0,0 +1,50 @@
+package net.atos.zac.mail
+
+import jakarta.mail.Address
+import jakarta.mail.Message
+import jakarta.mail.Session
+import jakarta.mail.internet.MimeBodyPart
+import jakarta.mail.internet.MimeMessage
+import jakarta.mail.internet.MimeMultipart
+import jakarta.mail.internet.PreencodedMimeBodyPart
+import net.atos.zac.mail.model.Attachment
+import java.nio.charset.StandardCharsets
+import java.util.Date
+
+class MailMessageBuilder(
+    private var fromAddress: Address,
+    private var toAddress: Address,
+    private var replyToAddress: Address? = null,
+    private var mailSubject: String,
+    private var body: String,
+    private var attachments: List<Attachment>
+) {
+    fun build(mailSession: Session) = MimeMessage(mailSession).apply {
+        addHeader("Content-type", "text/HTML; charset=${StandardCharsets.UTF_8.name()}")
```

Comment:
as far as I know "text/html" is always written in lowercase?

```suggestion
        addHeader("Content-type", "text/html; charset=${StandardCharsets.UTF_8.name()}")
```